samples/besu/simpleasset/README.md

```diff
@@ -0,0 +1,29 @@
+<!--
+ Copyright IBM Corp. All Rights Reserved.
+
+ SPDX-License-Identifier: CC-BY-4.0
+ -->
+# A Sample Application for Asset Exchange across Two Besu Networks using HTLC
+
+## Prerequisites
+- Truffle: npm install -g truffle
+
+## Deploy contracts
+The following command deploys the contracts on the two test networks:
+```
+make deploy-contracts
+```
+This internally does the following:
+- copies the core network contracts to the contracts folder in the truffle application. 
+	* In future, this should be replaced by exporting the core contracts as an npm or EthPM package and the app importing this package.
+	* The support is also only for ERC-20 tokens right now. This will be extended to ERC-721 soon. And in this future for any asset type.
+- updates truffle-config.js based on the environment variables BESU\_NETWORK\_HOST and BESU\_NETWORK\_PORT
+- installs the missing packages
+- compiles the contracts and migrates them to both the test networks
+
+## Run the asset exchange application
+The sample application will have Alice transfer AliceERC20 tokens in Network 1 to Bob atomically with Bob transfering an equivalent amount of BobERC20 tokens to Alice in Network 2.
+```
+node app/AssetExchangeERC20.js
```

Comment:
The one in Line 14 is sufficient. There is an `npm install` eventually run in `make deploy-contracts`.

Reply:
There used to be a problem with not installing Truffle separately. Since you are doing a fresh check, can you try without `npm install -g truffle`, because I also have Truffle as a dev dependency in package.json.

Reply:
I don't see an `npm install` in the `Makefile`.

Reply:
@VRamakrishna He has it in `package.json`.